### DISCUSSION TITLE
贡献另一个仓库的建议

### DISCUSSION THREAD

**Discussion by langningchen** [langningchen/XMOJ-Script#166] [Ideas]
@boomzero 我们可以一起贡献另一个仓库： https://github.com/langningchen/XMOJ

**Comment by langningchen** [comment]
或者： https://github.com/langningchen/Luogu

**Reply by boomzero** [reply]
你的run.sh为什么是用fish写的? 我只有bash和zsh

**Reply by boomzero** [reply]
哦我有fish

**Comment by boomzero** [comment]
https://github.com/langningchen/XMOJ/pull/1
